WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -137,6 +137,12 @@ def users_to_sentence(users, include_email: false)
     end.xss_aware_to_sentence
   end
 
+  def year_option_tags(selected_year: nil, future: false)
+    years = future ? Competition.years : Competition.non_future_years
+
+    content_tag(:option, t('competitions.index.all_years'), value: 'all years') + options_for_select(years, selected_year)
```

Comment:
Ah! I think that's been working by luck because it's the first thing and if nothing is marked as selected, then the first item ends up selected. We have the same weirdness in `def region_option_tags` below, which is what I copied when writing this method.

I've changed change both of them to be more explicit in https://github.com/jfly/worldcubeassociation.org/commit/634b8f6514a38f73cbcf5c76faf30d89e2f79543.